app/controllers/application_controller.rb

```diff
@@ -90,10 +91,4 @@ def prompt_to_set_up_2fa
   def prompt_to_enter_otp
     redirect_to user_two_factor_authentication_url
   end
-
-  def track_get_requests
```

Comment:
It does make sense, thanks for clarifying.

We will need to update the parameter filtering to reflect the IdV process then, because all the PII that happens there is not filtered currently. I will open a PR for that.